docs/usage_guide.rst

```diff
@@ -593,25 +593,45 @@ The list of currently supported airdrops is:
 Manually adding trades or other events
 **************************************
 
+Adding historical events
+========================
+
 Rotki will pull all your trade history from the exchanges whenever it needs it. But most of us have probably also done some OTC trades or taxable events at some point. Such events could even just be mining tokens, depending on your jurisdiction, participating in an ICO or getting paid in crypto.
 
-On the left sidebar click on the trades button and select "OTC Trades" from the dropdown menu. This will take you to the OTC Trades page.
+On the left sidebar click on History and then the Trade button from the dropdown menu. This will take you to the Trades page. Clicking on the + symbol will open you a menu as the following one
 
-.. image:: images/sc_add_external_trade.png
+.. image:: images/external_trade.png
    :alt: Add an external trade
    :align: center
 
-To add a new trade or taxable event, fill in all the fields and press the "Add Trade" button.
+To add a new trade or taxable event, fill in all the fields and press the "Save" button.
 
-Some very important things to note. All pairs should be in the form of ``BASECURRENCY_QUOTECURRENCY``. For a ``buy`` this means you are buying ``amount`` of the ``BASE`` currency at a price of ``rate`` ``QUOTE`` currency per ``BASE``. For a ``sell`` this means you are selling ``amount`` of the ``BASE`` currency at a price of ``rate`` ``QUOTE`` currency per ``BASE``.
+If there was a fee for the trade you should input it in the corresponding box and also enter the currency the fee was paid in. This field is optional so if the Fee was 0 you can leave this field empty. You can optionally provide additional notes or even links to blockchain explorers for each trade.
 
-If there was a fee for the trade you should input it in the corresponding box and also enter the currency the fee was paid in. Fee can also be 0.
+In the Trade page you can see a table of all your OTC trades. You can edit or delete a trade by clicking on the appropriate icon at the rightmost part of each trade under the "Actions" column.
 
-You can optionally provide additional notes or even links to blockchain explorers for each trade.
+Currently rotki tracks your balance querying the different supported protocols, exchanges and blockchains. If you manually add information about a trade your balances won't be updated since trades are not consulted when updating the accounts' balances. If after adding a trade you want to update the accounts balances please refer to the :ref:`manual balances section <track_balances>`.
```

Comment:
Nice note. This also definitely (along with the other common questions) needs to be part of the FAQ. Small rewording here:

```suggestion
Currently rotki tracks your balance by querying the different supported protocols, exchanges and blockchains. If you manually add information about a trade your balances will not be updated since trades are not consulted when updating the accounts' balances. If you want to manually update your balance for an asset please refer to the :ref:`manual balances section <track_balances>`.
```